smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/HttpProtocolTestGenerator.java

```diff
@@ -433,9 +449,50 @@ private void writeParamAssertions(Shape operationOrError, HttpResponseTestCase t
                     .call(() -> params.accept(new CommandOutputNodeVisitor(testOutputShape)))
                     .write("][0];");
 
+            // Extract a payload binding if present.
+            Optional<HttpBinding> payloadBinding = operationOrError.asOperationShape()
```

Comment:
Let's take an action item in the future to see if we can make this a little easier to do. Like imagine something like this method, but that could return a single optional that contains the first element if any are found? https://github.com/awslabs/smithy/blob/master/smithy-model/src/main/java/software/amazon/smithy/model/knowledge/HttpBindingIndex.java#L222